Automattic/sensei
# Add usage tracking for active and completed courses

This PR logs the following usage tracking data:

- Total number of active courses across all learners
- Total number of completed courses across all learners

## Testing
1. Check that `course_completed` is correctly logged in Tracks. Completed data should match what shows in _Sensei_ > _Analysis_ > _Total Completed Courses_.
2. Check that `course_active` is correctly logged in Tracks. Active data can be found in in _Sensei_ > _Analysis_ by summing the values in the _Active Courses_ column.
